liquid/liquid.go

```diff
@@ -0,0 +1,95 @@
+package liquid
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/json"
+	"errors"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"time"
+
+	"github.com/VictoriaMetrics/fastcache"
+	"github.com/breez/server/breez"
+)
+
+func checkSwapId(fc *fastcache.Cache, apiURL, swapID string, body []byte) error {
+	c := &http.Client{Timeout: 2 * time.Second}
```

Comment:
Seems too short which might cause broadcasts to fail. I would go with 10 seconds.